view/footer/index.css

```diff
@@ -0,0 +1,120 @@
+.footer {
+    z-index: 999;
```

Comment:
Let's try to use small values (-1, 0, 1) or a little more? :)

Reply:
done)